sipa/static/css/style.css

```diff
@@ -168,3 +168,34 @@ img[alt=software_logo] {
 tr:hover .text-password {
 	filter: none;
 }
+
+.red {
+	color: red;
```

Comment:
It might be more consistent to use bootstrap's own color scheme here: Either
- via CSS variables (https://getbootstrap.com/docs/5.0/customize/css-variables/#root-variables)
- or via a [utility class](https://getbootstrap.com/docs/5.0/utilities/colors/#colors) (`.text-red`)

Reply:
> or via a [utility class](https://getbootstrap.com/docs/5.0/utilities/colors/#colors) (.text-red)

`text-red` is not available but `text-danger` is.

Also note we're still using Bootstrap 3.3.5 :

https://github.com/agdsn/sipa/blob/bd8ed8577196f9d6347628c4873a92090ac619dc/sipa/static/css/bootstrap.min.css#L2

Reply:
> Also note we're still using Bootstrap 3.3.5 :

Nevermind then.